config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -28,10 +31,9 @@ presubmits:
       testgrid-num-failures-to-alert: '10'
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
-  # TODO(krzyzacy): add branches once this works
-  # TODO(krzyzacy): Consider combine kubekins-e2e and kubekins-test
-  - name: pull-kubernetes-integration-podutil
-    always_run: false
+  # TODO: replace the bootstrap / scenario based job going forward
```

Comment:
do we need to change the `testgrid-...` annotations?

Reply:
yeah

Reply:
done, I compared to `-verify`, we don't need any annotations on this one afaict, it will leverage the annotations on the other entry